content/docs/guides/vercel-app.md

```diff
@@ -0,0 +1,378 @@
+
+    ```bash
+    wget https://github.com/neondatabase/naturesnap/archive/refs/heads/main.zip
+    unzip main.zip
+    rm main.zip
+    mv naturesnap-main naturesnap
+    ```
+
+2. Install the dependencies in the project's root directory:
+
+    ```bash
+    npm install
+    npm install dotenv
+    ```
+
+3. Create a Neon project.
+    1. Navigate to the [Neon Console](https://console.neon.tech).
+    1. Select **Create a project**.
+    1. Enter `naturesnap` as the project name and select a PostgreSQL version and a region
+    1. Click **Create project**.
+
+    ![Create naturesnap project](/docs/guides/ns_create_project.png)
+
+    Upon creating the project, you are presented with a dialog that provides connection details for your default project user. You will create a different user for the application, so you do not have to save this information.
+4. Create a database user named `naturesnap`, which the application uses to access the `naturesnap` application database. In the [Neon Console](https://console.neon.tech):
+    1. Select **Users**.
+    1. Select **New User**.
+    1. Enter `naturesnap` as the user name.
+    1. Click **Create**.
+    ![Create naturesnap user](/docs/guides/ns_create_user.png)
+    1. Upon creating the user, you are presented with a dialog that provides connection details for the user. Save the password. It is required in a later step.
+5. Create a database for the application and call it it `naturesnap`. In the [Neon Console](https://console.neon.tech):
+    1. Select **Databases**.
+    1. Click **New Database**.
+    1. Enter  `naturesnap` as the database name.
+    1. Select `naturesnap` as the database owner.
+    1. Click **Create**.
+    ![Create naturesnap database](/docs/guides/ns_create_app_db.png)
+6. Create a shadow database for Prisma Migrate. In the [Neon Console](https://console.neon.tech):
+    1. Select **Databases**.
+    1. Click **New Database**.
+    1. Enter  `shadow` as the database name.
+    1. Select `naturesnap` as the database owner.
+    1. Click **Create**.
+    ![Create shadow database](/docs/guides/ns_create_shadow_db.png)
+7. Create a `.env` file at the root of your project directory and add the `DATABASE_URL` and `SHADOW_DATABASE_URL` environment variables and set the values to the URLs for the `naturesnp` and `shadow` databases. Prisma Migrate requires these settings to migrate the database schema to the databases. You retrive the connection strings from the **Connection Details** widget on the Neon Dashboard. Don't forget to add the password to the connection string for the `naturesnap` user you created previously.
+
+    ```bash
+    cd naturesnap
+    touch .env
+    ```
+
+    When finished, your `.env`  file should appear similar to the following (your hostname will differ).
+
+    ```text
+    DATABASE_URL=postgres://naturesnap:************@ep-snowy-water-747999.us-east-2.aws.neon.tech/naturesnap?sslmode=require&connect_timeout=0
```

Comment:
I think for populating local envs we should advice users to user development branches

Reply:
The point of the integration is so, that user doesn't need to manage neon project manually